docker/docker-compose.yml

```diff
@@ -39,7 +39,7 @@ services:
       - DATABASE_URL=postgresql://cuttlesworth:p4ssw0rd!@database:5432/cuttle
       # Can set this to staging for the postgres database to work in docker but may need to modify
       # the onlyAllowOrigins setting in `config/env/staging.js` file
-      - NODE_ENV=staging
+      - NODE_ENV=development
```

Comment:
Switched to development so that we use the mock db which currently works better for testing in CI